src/pocketmine/item/Item.php

```diff
@@ -821,6 +821,17 @@ public function setCount(int $count){
 		return $this->canBePlaced();
 	}
 
+	public function canBeConsumed() : bool{
```

Comment:
My point is not about the modifier. I'm talking about how it should be used. Even if they are improper, you can still add an `@internal` JavaDoc tag **if it is useful**.